CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.0.2](https://github.com/microsoftgraph/microsoft-graph-toolkit/compare/v4.0.1...v4.0.2) (2024-02-13)
+
+
+### Bug Fixes
+
+* add permissions for CodeQL ([#3041](https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/3041)) ([f84d6ef](https://github.com/microsoftgraph/microsoft-graph-toolkit/commit/f84d6ef96dd126f1b2bff381442efce796043f80))
```

Comment:
This isn't a bug fix, but rather a chore. adding a blocking comment until we have more commits that are releasable units to correct this entry in the change log as I've added an override to the source PR